lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -3791,26 +3794,23 @@ LogicalResult FIRRTLLowering::visitExpr(LTLClockIntrinsicOp op) {
                                         getLoweredNonClockValue(op.getClock()));
 }
 
-LogicalResult FIRRTLLowering::visitExpr(LTLDisableIntrinsicOp op) {
-  return setLoweringToLTL<ltl::DisableOp>(
-      op,
-      ValueRange{getLoweredValue(op.getLhs()), getLoweredValue(op.getRhs())});
-}
-
 LogicalResult FIRRTLLowering::visitStmt(VerifAssertIntrinsicOp op) {
   builder.create<verif::AssertOp>(getLoweredValue(op.getProperty()),
+                                  getLoweredValue(op.getEnable()),
```

Comment:
nit: Can we avoid calling (potentially) side-effecting functions in function arguments since their evaluation order is undefined? 

getLoweredValue
```suggestion
auto property = getLowerdValue(op.getProperty());
auto enable = op.getEnable() ? getLoweredValue(op.getEnable()): Value();
builder.create<verif::AssertOp>(property, enable);
```